netmaster/k8snetwork/networkpolicy.go

```diff
@@ -66,18 +146,26 @@ func (k8sNet *k8sContext) createNetwork(nwName string) error {
 	}
 
 	for func() error {
-		_, err := k8sNet.contivClient.NetworkGet(defaultTenantName, nwName)
+		_, err := k8sNet.contivClient.NetworkGet(
+			defaultTenantName,
+			nwName)
 		return err
 	}() != nil {
+		//XXX:Should we really poll here ;
```

Comment:
if an infinity loop is here ( without enough logs and hints), no one would able to know how to find out.
please put logs and a maxretry here.

Reply:
agree, will put logs with timeout